src/vm/gcenv.ee.cpp

```diff
@@ -1176,13 +1176,13 @@ void GCToEEInterface::DiagWalkFReachableObjects(void* gcContext)
 // don't get confused.
 void WalkMovedReferences(uint8_t* begin, uint8_t* end, 
                          ptrdiff_t reloc,
-                         size_t context, 
-                         BOOL fCompacting,
-                         BOOL fBGC)
+                         void* context, 
+                         bool fCompacting,
+                         bool fBGC)
```

Comment:
yeah, that's why I said not *strictly* :-) either we change this and change the callers too; or we leave this unchanged but I wouldn't want it to be in an inbetween state.